src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -418,6 +419,13 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
         },
         ov::pass::NormalizeL2Decomposition);
 
+    // todo: open this callback to go with snippets after reduce decompose merged
```

Comment:
Maybe I missed something but GroupNormalization is tokenized only if it's static node. Seems like this comment is outdated. Moreover, I'd suggest to remove all block of the code here

Correct me please if I'm wrong.

Reply:
CPU plugin does not implement GroupNormalization node on current master. The GroupNorm is decomposed to mvn and other ops in common optimization, so that it can be supported. While we support static GroupNorm via snippets. we should not decompose it here if static shape. In the later transformation, it will be tokenized to subgraph and decomposed in snippets pipeline. This "GroupNormalizationDecomposition" is not my new added pass. I changed new pass as "GNDecomposition" to distinguish.

Reply:
Thanks for the explanation!
At first time I didn't notice that it's callback for common transformation. I thought it's for snippets pipeline 😄 It's my mistake.